src/_data/platforms.yml

```diff
@@ -267,8 +267,7 @@
       doc_link: /clients/python/integrations/bottle/
```

Comment:
This is still referring to the old Raven SDK while your updated lines refer to the new client.

It should be `/platforms/python/bottle/`

Reply:
Fixed. Thanks for noticing.